components/Calculates.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+
+export default () => {
+  const [numberOne, setNumberOne] = useState('')
+
+  const [numberTwo, setNumberTwo] = useState('')
+  const [setOperators] = useState('')
+  const [result, getResult] = useState('')
+
+  return (
+    <div className="form">
+      <div className="intro">React Calculator</div>
+      <input type="text" id="numberOne" onChange={event => setNumberOne(event.target.value)} value={numberOne} />
+      <button type="button" id="operators" onClick={() => { setOperators('') }}>
+        <select id="operators">
+          <option value="+">+</option>
+          <option value="-">-</option>
+          <option value="*">*</option>
+          <option value="/">/</option>
+        </select>
+      </button>
+      <input type="text" id="numberTwo" onChange={event => setNumberTwo(event.target.value)} value={numberTwo} />
+      <button type="button" onClick={() => { getResult('') }}>=</button>
+      <input type="text" id="result" onChange={event => getResult(event.target.value)} value={result} />
+      <div className="error">
+        { numberOne || numberTwo ? { numberOne, numberTwo } : 'Please provide a valid number for both operands'}
+      </div>
```

Comment:
This is what is throwing the error, you are saying that if numberOne or numberTwo has a value to put an object made up of them on the screen. You can't put an object on the screen but also that doesn't make sense. You'd want to put the error message on the screen if either is not a number, and put nothing otherwise.

![image](https://user-images.githubusercontent.com/5775716/83420796-20278f80-a3f5-11ea-8d07-d7c9f1b777a7.png)